source/includes/_resource.videos.md

```diff
@@ -224,6 +229,26 @@ vhx.videos.create({
       must be a url of an image, hosted on vhx, prefixed with the text <code>image_url:</code>. 
       </td>
     </tr>
+
+    <tr class="is-internal text-2 border-bottom border--light-gray">
+      <td>
+        <strong class="is-block text--navy">plans</strong>
+        <span class="text--transparent text-3">array of strings</span>
+      </td>
+      <td>
+      An array of plan types that you can use to set video availability. 
+      Values can be one or more of the following strings: <code>public</code>, <code>free</code>,
+      <code>standard</code>.<br><br>
+
+      The <code>public</code> plan makes the video object available for free, without registration 
+      to any public visitor to your site.<br><br>
```

Comment:
"The public plan makes the video object available without email registration or paid subscription." ?